emily/handler/src/context.rs

```diff
@@ -73,4 +73,66 @@ impl EmilyContext {
             dynamodb_client: Client::new(&config),
         })
     }
+    /// Create a local testing instance.
+    #[cfg(feature = "testing")]
+    pub async fn local_test_instance() -> Result<Self, Error> {
+        use std::collections::HashMap;
+
+        // Get config that always points to the dynamodb table directly
+        // from outside of a docker compose setup.
+        let dynamodb_client = Client::new(
+            &aws_config::load_defaults(BehaviorVersion::latest())
+                .await
+                .into_builder()
+                .endpoint_url("http://localhost:8000")
+                .build(),
+        );
+
+        // Get the names of the existing tables so we can populate from them.
+        let table_names = dynamodb_client
+            .list_tables()
+            // Get at most 20 table names - there should be 3...
+            .limit(20)
+            .send()
+            .await
+            .map_err(|err| {
+                Error::Debug(format!(
+                    "Failed setting up settings from table names - {err:?}"
+                ))
+            })?
```

Comment:
I believe what's here is equivalent to `.expect("Failed setting up settings from table names")`